### PR TITLE
translation: Update chapter_stack_and_queue/index.md

### PR DESCRIPTION
**Pull request by YangXuanyi**
If this PR is related to coding or code translation, please fill out the checklist and paste the console outputs to the PR.

- [ ] I've tested the code and ensured the outputs are the same as the outputs of reference code.
- [ ] I've checked the code (formatting, comments, indentation, file header, etc) carefully.
- [ ] The code does not rely on a particular environment or IDE and can be executed on a standard system (Win, macOS, Ubuntu).

### CODE REVIEW

**Comment by krahets** [comment]
Randomly assign 2 reviewers ['@K3v123' '@yadie'] to  the pull request #1035

**Review by K3v123 — APPROVED** [review]

**Comment by K3v123** [comment]
looks good to me. Although I do prefer the original on the second sentence, but both are good anyways.

**Comment by krahets** [comment]
> Although I do prefer the original on the second sentence, but both are good anyways.

Me too

**Comment by YangXuanyi** [comment]
@krahets @K3v123 Thank you for your opinions. I personally think the second sentence I provided may be more detailed. Since you all agree that both methods are fine, I'm going to use my own modification of this one.

File: docs-en/chapter_stack_and_queue/index.md [hunk 1]
```diff
@@ -8,6 +8,6 @@
 
 !!! abstract
 
-    Stacks are like stacking cats, while queues are like cats lining up.
+    A stack is like cats placed on top of each other, while a queue is like cats lined up one by one.
     
-    They respectively represent the logical relationships of Last-In-First-Out (LIFO) and First-In-First-Out (FIFO).
+    They represent the logical relationships of Last-In-First-Out (LIFO) and First-In-First-Out (FIFO), respectively.
```
Comment (yanedie) [inline docs-en/chapter_stack_and_queue/index.md:13]: 不需要逗号。

**Review by yanedie — CHANGES_REQUESTED** [review]

File: docs-en/chapter_stack_and_queue/index.md (hunk 1 above)
Comment (YangXuanyi) [reply docs-en/chapter_stack_and_queue/index.md:13]: 非常感谢您的建议。这个逗号的问题我在翻译的时候也查阅了一些资料，我发现从语法的角度上来说加和不加都可以，但是看到一些科技论文里大部分都是加上的。您提到这个后我也不确定了，可以请详细阐述为什么要删除逗号吗？想交流学习下这个词的用法

Comment (yanedie) [reply docs-en/chapter_stack_and_queue/index.md:13]: 抱歉，我在各个字典里看到的例句都是不加逗号的，然后我查阅了各个杂志期刊发现也有加的。

Comment (yanedie) [reply docs-en/chapter_stack_and_queue/index.md:13]: > 非常感谢您的建议。这个逗号的问题我在翻译的时候也查阅了一些资料，我发现从语法的角度上来说加和不加都可以，但是看到一些科技论文里大部分都是加上的。您提到这个后我也不确定了，可以请详细阐述为什么要删除逗号吗？想交流学习下这个词的用法

抱歉，我在各个字典里看到的例句都是不加逗号的，然后我查阅了各个杂志期刊发现也有加的。

**Review by yanedie — APPROVED** [review]

**Review by krahets — APPROVED** [review]

**Pull request merged**